app/src/main/java/com/github/se/icebreakrr/MainActivity.kt

```diff
@@ -49,6 +50,7 @@ fun IcebreakrrApp() {
         route = Route.AROUND_YOU,
     ) {
       composable(Screen.AROUND_YOU) { AroundYouScreen(navigationActions) }
+      composable(Screen.FILTER) { FilterScreen(navigationActions) }
```

Comment:
Remark : There is also a button filter in the Notification page, so idk if filter is considered in the workflow of the AroundYou Screen or the notification screen, we can keep it like this for now

Reply:
We'll have to talk about it with the whole team, I'm not sure the filter will be available on the inbox page.